Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py

```diff
@@ -0,0 +1,806 @@
+        as a bytes object four characters in length.
+
+        This is specifically used for creating scan payloads when sending POST requests using
+        our FrontlineVM API within the build_scan method.
+    '''
+    return struct.unpack("!L", socket.inet_aton(ip_address))[0]
+
+
+def ip_number_to_address(ip_number):
+    '''
+        Convert a 32-bit packed IPv4 address (a bytes-like object four bytes in length)
+        to its standard dotted-quad string representation.
+
+        This is specifically used for creating scan payloads when sending POST requests using
+        our FrontlineVM API within the build_scan method.
+    '''
+    return socket.inet_ntoa(struct.pack("!L", ip_number))
+
+
+def get_network_data():
+    ''' Get network data. Used to perform scan. '''
+    try:
+        url = BASE_URL + "/api/networkprofiles/?_0_eq_networkprofile_internal=True"
+        resp = requests.get(url, headers=API_AUTH_HEADER)
+        resp.raise_for_status()
+        return json.loads(resp.text)
+    except Exception as err:
+        return_error('Error: getting network data -- ' + str(err))
+    return []   # placed to satisfy pylint (inconsistent-return-statements error)
+
+
+def get_scan_data(network_data, low_ip, high_ip):
+    ''' Iterate through network data to find appropriate scanner profile to use to perform scan.'''
+    for profile in network_data:
+        # If there is no scanner using this profile, then continue to the next profile
+        if len(profile.get('scanner_names', "")) == 0:
+            continue
+        scanner_id = profile.get('scanner_ids')[0]
+        scanner_url = BASE_URL + "/api/scanners/" + str(scanner_id) + "/"
+        scanner_resp = requests.get(scanner_url, headers=API_AUTH_HEADER)
+        scanner_resp.raise_for_status()
+        scanner = json.loads(scanner_resp.text)
+        if scanner.get('status', '') == 'online':
+            url = BASE_URL + "/api/networkprofiles/" + str(profile['id']) + "/rules/"
+            profile_data = []   # type: List
+            have_all_data = False
+            while not have_all_data:
+                resp = requests.get(url, headers=API_AUTH_HEADER)
+                resp.raise_for_status()
+                current_data = json.loads(resp.text)
+                profile_data.extend(current_data.get('results', []))
+                if current_data.get('next', None):
+                    url = current_data.get('next')
+                else:
+                    have_all_data = True
+            for rule in profile_data:
+                if rule.get('ip_address_range', None):
+                    rule_high_ip_num = rule['ip_address_range']['high_ip_number']
+                    rule_low_ip_num = rule['ip_address_range']['low_ip_number']
+                    if (rule_high_ip_num >= high_ip) and (rule_low_ip_num <= low_ip):
+                        return {'profile_id': profile['id']}
+    return_error("Error: no scan data found.")
```

Comment:
The reason for the error is because without a scanner profile (no scan data found), a scan can not be perform, thus stopping the scan-asset command.
Removed `removed_error()` and replaced with `demisto.results()`

Reply:
Is there a way we can stop the scan asset command at this point and display to the user that no scanner profile has been found to scan the asset?